NumLib/DOF/ComputeSparsityPattern.cpp

```diff
@@ -53,19 +53,11 @@ GlobalSparsityPattern computeSparsityPatternNonPETSc(
     // Map adjacent mesh nodes to "adjacent global indices".
     for (std::size_t n = 0; n < mesh.getNumberOfNodes(); ++n)
     {
-        auto const& node_ids = node_adjacency_table.getAdjacentNodes(n);
-        for (auto an : node_ids)
-        {
-            auto const& row_ids = global_idcs[an];
-            auto const num_components = row_ids.size();
-            for (auto r : row_ids)
-            {
-                // Each component leads to an entry in the row.
-                // For the sparsity pattern only the number of entries are
-                // needed.
-                sparsity_pattern[r] += num_components;
-            }
-        }
+        unsigned n_connected_dof = 0;
```

Comment:
a question: do we only consider the case of  NumLib::ComponentOrder::BY_COMPONENT for HM?